.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -159,7 +159,11 @@ jobs:
             --set=clustermesh.useAPIServer=${{ !matrix.external-kvstore }} \
             --set=clustermesh.maxConnectedClusters=${{ matrix.max-connected-clusters }} \
             --set=clustermesh.config.enabled=true \
-            --set=extraConfig.clustermesh-ip-identities-sync-timeout=10m"
+            --set=extraConfig.clustermesh-ip-identities-sync-timeout=10m \
+            --set=clustermesh.apiserver.readinessProbe.periodSeconds=1 \
+            --set=clustermesh.apiserver.kvstoremesh.readinessProbe.periodSeconds=1 \
+            --set=clustermesh.apiserver.updateStrategy.rollingUpdate.maxSurge=1 \
+            --set=clustermesh.apiserver.updateStrategy.rollingUpdate.maxUnavailable=0"
```

Comment:
Nit. I'd add a comment to the list above about the reasons for configuring these settings.